ansible_collections/netapp/ontap/changelogs/fragments/20.10.0.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - na_ontap_lun - ``use_exact_size`` to create a lun with the exact given size so that the lun is not rounded up
```

Comment:
can you revert this change?   You should not modify this file, and you already created a fragments/github-56.yaml

Reply:
Deleted the file.